logilica_weekly_report/cli_weekly_report.py

```diff
@@ -111,12 +124,30 @@ def weekly_report(
         pdf_items = get_pdf_objects(
             teams=configuration["teams"], download_dir_path=download_dir_path
         )
-        doc = generate_html(pdf_items)
-        if output == "gdoc":
-            url = upload_doc(doc.getvalue(), google_credentials, config)
-            click.echo(f"Report uploaded to {url}")
+        if output == "images-only":
+            output_dir_missing = not output_dir_path.exists()
+            logging.debug(
+                "output directory %s",
+                "does not exist" if output_dir_missing else "exists",
+            )
+            if output_dir_missing:
+                os.mkdir(output_dir_path)
+            for team, dashboards in pdf_items.items():
+                for dashboard, png in dashboards.items():
+                    imagepath = Path(output_dir_path) / f"{team}-{dashboard}.png"
```

Comment:
I believe that `output_dir_path` is already a `Path`, so there is no need to construct a new one from it (the `/` operator will do that for you...).

Reply:
Ack.